unishox2.c

```diff
@@ -73,6 +73,7 @@ const int UTF8_PREFIX[] = {0xC0, 0xE0, 0xF0};
 
 #define SW_CODE 0
 #define SW_CODE_LEN 2
+#define TERM_BYTE_PRESET_1 (byte)(SW_CODE | ((SW_CODE | ((SW_CODE | ((SW_CODE | ((SW_CODE | ((SW_CODE | ((SW_CODE | (SW_CODE >> SW_CODE_LEN)) >> SW_CODE_LEN)) >> SW_CODE_LEN)) >> SW_CODE_LEN)) >> SW_CODE_LEN)) >> SW_CODE_LEN) >> SW_CODE_LEN)))
```

Comment:
I don't understand the object of this.  Since SW_CODE is 0, won't OR or SHIFT still give 0 as a result?

Reply:
the purpose of TERM_BYTE_PRESET_1 is that, it should be defined as many continuous SW_CODE filing a whole byte, even if SW_CODE is defined as other bit sequence.

Reply:
ok, I understand, but I think SW_CODE will always be 00.  It is just "defined" for readability.